### PR TITLE
enable local zone and version for rosa

### PR DESCRIPTION
**Pull request by yasun1**

### CODE REVIEW

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-sts-advanced-f3

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-sts-advanced-f3

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-sts-advanced-f3

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-sts-advanced-f3

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-sts-advanced-f3

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-sts-advanced-f3

**Comment by yasun1** [comment]
@xueli181114 @yuwang-RH Please help review. Thanks.

File: ci-operator/step-registry/rosa/conf/machine-pool/rosa-conf-machine-pool-commands.sh [hunk 1]
```diff
@@ -12,7 +12,7 @@ MP_MACHINE_TYPE=${MP_MACHINE_TYPE:-"m5.xlarge"}
 MP_ENABLE_AUTOSCALING=${ENABLE_AUTOSCALING:-false}
 USE_TUNING_CONFIG=${USE_TUNING_CONFIG:-false}
 USE_SPOT_INSTANCES=${USE_SPOT_INSTANCES:-false}
-SPOT_MAX_PRICE=${SPOT_MAX_PRICE:-"0.1"}
+SPOT_MAX_PRICE=${SPOT_MAX_PRICE:-"on-demand"}
```
Comment (xueli181114) [inline ci-operator/step-registry/rosa/conf/machine-pool/rosa-conf-machine-pool-commands.sh:15]: Why hardcode to on-demand?

**Comment by xueli181114** [comment]
Not all regions support local zone, is the region hardcode?

File: ci-operator/step-registry/rosa/conf/machine-pool/rosa-conf-machine-pool-commands.sh (hunk 1 above)
Comment (yasun1) [reply ci-operator/step-registry/rosa/conf/machine-pool/rosa-conf-machine-pool-commands.sh:15]: It is the default value

**Comment by yasun1** [comment]
@xueli181114 The region is from the CLUSTER_PROFILE, to some degree, the region is hardcoded.

**Pull request merged**